ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -438,7 +442,10 @@ export function PreviewConcierge(props: any) {
               dispatch(setPreviewChoice({ site, choice: '2' }));
             }
 
-            getHostToGuestBus().next({ type: HOST_CHECK_IN, payload: { editMode, highlightMode } });
+            getHostToGuestBus().next({
+              type: HOST_CHECK_IN,
+              payload: { editMode: false, highlightMode }
```

Comment:
How come editMode is hard set to false here?

Reply:
we talked about this yesterday, guest is going to start false and the effect on the top of previewConcierge is calculating the getComputedEditMode, and updating Guest